File: Practice problem/Number of p and y in string.py
# 연습문제 문자열 내 p와 y의 개수 Level 1

def solution(s):
    count = 0
    for i in range(len(s)):
        if(s[i] == 'p' or s[i] == 'P'):
            count += 1
        elif(s[i] == 'y' or s[i] == 'Y'):
            count -= 1
    if(count == 0):
        return True
    else:
        return False

File: Practice problem/Number of p and y in string.cpp
// 연습문제 문자열 내 p와 y의 개수 Level 1

#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    int count = 0;
    for(int i = 0; i < s.length(); i++){
        if(s.at(i) == 'p' || s.at(i) == 'P'){ count++; }
        else if(s.at(i) == 'y' || s.at(i) == 'Y'){ count--; }
    }
    if(count == 0){ return true; }
    else{ return false; }
}
